app/src/test/java/com/boarbeard/generator/beimax/BogoSharedPreferences.java

```diff
@@ -0,0 +1,100 @@
+package com.boarbeard.generator.beimax;
+
+import android.content.SharedPreferences;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
+import androidx.annotation.Nullable;
+
+import static org.junit.Assert.fail;
+
+/**
+ * An incredibly dumb SharedPreferences implementation which probably didn't
+ * need to be written.  Make that <i>definitely</i> didn't need to be written.
+ */
+public class BogoSharedPreferences implements SharedPreferences {
```

Comment:
we can just use androidx sharedpreferences at some point, but yea this is great till then